react-ui/src/pages/contest/Contest.jsx

```diff
@@ -564,7 +568,8 @@ function Contest() {
                         className={clsx(
                           classes.categoryChip,
                           categoryLabelClasses[
-                            `label${categories.indexOf(value) % LABEL_COLORS.length}`
+                          // eslint-disable-next-line indent
```

Comment:
Was this line giving you problems?

Reply:
Ya, it was a fight. I can also do `// prettier-ignore` if you prefer.